README.md

```diff
@@ -1,29 +1,32 @@
 # Pozter
-
 Applied Research Winter 2022 Project Pozter
-
-# Project setup:
```

Comment:
Please leave the previous readme details and add more if you'd like. 
It looks like you add your pull request details into the readme which shouldn't be here.